targets/Program.cs

```diff
@@ -131,7 +131,7 @@ static void CleanDirectory(string directoryPath)
                 }
             }
             catch (AccessViolationException) { /* swallow */ }
-            // catch (Exception ex) { Console.WriteLine(ex.Message); }
+            catch (IOException) { Directory.Delete(directoryPath, true); /* try again */ }
```

Comment:
What's this?

Reply:
whoops, something I was testing out, I was running into an issue where some file handles were sticking around longer than I wanted. not actually needed, I think